content/02.introduction.md

```diff
@@ -1,7 +1,9 @@
+The degraded nature of aDNA poses an extra layer of complexity to standard modern genomic analysis. Through a variety of processes [@doi:10.1038/362709a0], DNA molecules will fragment overtime resulting in ultra-short molecules (e.g. down to 35 bp [@doi:10.1038/nature17405]). These sequences low nucleotide complexity making it difficult identify with precision which part of the genome a read is derived from. Furthermore, when fragmentation is not a 'clean break' this can lead to uneven ends with single-stranded 'overhangs' at end of molecules, which are susceptible to chemical processes such as deamination that are misread by some polymerases [@doi:10.1073/pnas.0704665104], resulting in incorrect base calling during sequencing. On top of this, taphonomic processes such as heat, moisture, and microbial and burial environment processes leads to varying rates of degradation [@doi:10.1093/nar/gkx361, @doi:10.1146/annurev-genom-091416-035526], where the original DNA content of a sample is loss and supplanted by modern environmental DNA. Later handling by modern archaeologists, museum curators, and scientists can also contribute 'modern' contamination. While these characteristics can help provide evidence towards the 'authenticity' of true aDNA sequences (e.g. aDNA C>T 'damage' profiles [@doi:10.1093/bioinformatics/btt193]), they also pose specific challenges such as unspecific DNA alignment and/or low coverage and miscoding lesions resulting in low-confidence genotyping. To retrieve enough data for modern NGS data pipelines these factors often lead to prohibitive sequencing costs (e.g. > 1 billion reads for a 1X depth coverage _Y. pestis_ genome [@doi:10.1016/j.cell.2015.10.009]), and thus require aDNA requires tailored methods and techniques to overcome these challenges.
```

Comment:
> These sequences low nucleotide complexity making it difficult identify with precision which part of the genome a read is derived from.

Missing word ?

These sequences **may have** low nucleotide complexity making it difficult identify with precision which part of the genome a read is derived from.

Reply:
> To retrieve enough data for modern NGS data pipelines these factors often lead to prohibitive sequencing costs (e.g. > 1 billion reads for a 1X depth coverage _Y. pestis_ genome [@doi:10.1016/j.cell.2015.10.009]), and thus require aDNA requires tailored methods and techniques to overcome these challenges.

these factors often lead to prohibitive sequencing costs to retrieve enough data for modern NGS data pipelines (e.g. > 1 billion reads for a 1X depth coverage _Y. pestis_ genome [@doi:10.1016/j.cell.2015.10.009]), and thus require aDNA requires tailored methods and techniques to overcome these challenges.